apps/condo/domains/property/components/panels/Builder/MapConstructor.ts

```diff
@@ -633,6 +634,14 @@ class MapEdit extends MapView {
         this.mode = mode
     }
 
+    get hasPreviewComponents (): boolean {
+        return !isNull(this.previewSectionId)
```

Comment:
can `preview*Id` be undefined? Mb use isNil?

Reply:
No it can not. It initial value is null and when it become cleared it also sets to null

Reply:
@nomerdvadcatpyat make this fields private with public getters. So it can be modified only from `MapConstructor` class methods